include/votca/tools/graph.h

```diff
@@ -57,15 +60,15 @@ class Graph : public EdgeContainer {
 
  public:
   Graph() : id_(""){};
-  ~Graph(){};
+  virtual ~Graph(){};
```

Comment:
you want these classes to be overloaded?

Reply:
e.g. inherited from

Reply:
Yes, you can let me know if this is a bad idea.

Reply:
Should I also delete the copy constructor for this Class?

Reply:
Never mind you already answered that

Reply:
no, might be a good idea. I do not know the use case yet.